brave/brave-core
# Adds check for public key when getting tokens

Resolves https://github.com/brave/brave-browser/issues/6939

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (unit, browser, security tests) on

## Test Plan:
- enable rewards
- claim grant
- make sure that balance updates correctly

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
